runtime/common/src/test/kotlin/kotlinx/serialization/json/polymorphic/JsonListPolymorphismTest.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.json.polymorphic
```

Comment:
Tests in this folder are run only for JVM and JS. Consider moving all tests to `testsWithNative`, which is eventually will be the main test folder (and renamed back to `test`). To do this, you'll need to pass serializer explicitly in some cases,  because `@ImplicitReflectionSerializer` is not available on Native. You can comment out build.gradle#43, reimport project and IDE will highlight to you problematic functions.